compiler/AST/AstDumpToNode.cpp

```diff
@@ -1875,9 +1875,9 @@ void AstDumpToNode::writeSymbol(Symbol* sym) const
       if (var->type == dtBool)
       {
         if (var->immediate->v_bool == 0)
-          sprintf(imm, "false");
+          snprintf(imm, bufSize, "false");
```

Comment:
Minor nitpick, but above you use `sizeof` even if the buffer size is also available (roughly, you've got `bufSize+1`). I think I like `sizeof` because even if we somehow decide "hey, I want +1 here too", the code doesn't break.

Reply:
I believe this decision is consistent. In the above instance with name, the buffer size is set to a magic constant, which could be changed to [magic constant]+1 or anything else as you say. However in this instance we have a constant created explicitly to store buffer size, so if it were to change it would change everywhere, and I find sizeof redundant.